### PR TITLE
[docs] add status header for kafka receiver

### LINKED ISSUE #12095: Add status header for kafka

### PR DESCRIPTION
**Pull request by codeboten**
Fixes #12095

Pinging codeowners to review status @pavolloffay @MovieStoreGuy

### CODE REVIEW

**Review by MovieStoreGuy — APPROVED** [review]

**Review by mx-psi — APPROVED** [review]

**Comment by mx-psi** [comment]
We have a codeowner approval, merging...

**Pull request merged**
